README.md

```diff
@@ -81,8 +84,34 @@ Se conheces uma comunidade não listada aqui, podes enviar um PR, que será de m
 
 <details>
   <summary>Ver detalhes</summary>
-   Via de Comunicação 
+   Via de Comunicação
 
   1. [Whatsapp](https://chat.whatsapp.com/ILHtN728Hhp1St5Ag9eDji)
 
 </details>
+
+---
+
+### Python Community Maputo
+
+1. [Whatsapp](https://chat.whatsapp.com/AtlP04I9rTL1GTbHPHKe3P)
+
+---
+
+### Js Maputo Community
+
+1. [Whatsapp](https://chat.whatsapp.com/1ZXaVqxGSM99MbMbnqPtur)
```

Comment:
@RichaldoElias is this correct? Shouldn't we use the `details` tag?

Reply:
Yeah I might have forgotten to add this one

Reply:
done here: https://github.com/RichaldoElias/ComunidadesMoz/pull/6